src/test/java/com/puppycrawl/tools/checkstyle/checks/AvoidEscapedUnicodeCharactersCheckTest.java

```diff
@@ -352,6 +356,16 @@ public void testAllowEscapesForControlCharacterSetForAllCharacters() throws Exce
         verify(checkConfig, getPath("InputAllEscapedUnicodeCharacters.java"), expected);
     }
 
+    @Test
+    public void testCountMatches() throws Exception {
```

Comment:
please explain in javadoc comment over method why it is not possible to make a functional test.
reflection usage should be avoided as much as possible.